### DISCUSSION TITLE
firefox addons page is gone

### DISCUSSION THREAD

**Discussion by aiwafag** [dimdenGD/OldTwitter#752] [Q&A] [answered]
so i uninstalled the firefox add-on, but came back because i noticed how much current twitter design sucks, and to my surprise, the firefox add-on page is gone.
i don't know if you guys know this, if you guys removed it or if mozilla removed it, but the page returns this error.
(i'm using the latest version of firefox)
![Screenshot_2](https://github.com/dimdenGD/OldTwitter/assets/167035969/362e163b-3acb-4121-ae82-5f8b34e1fceb)

**Comment by dimdenGD** [comment, marked as answer]
https://twitter.com/d1mden/status/1783854522663665783

**Reply by aiwafag** [reply]
thank you for the info dear fellow azumanger

**Reply by Salty4life** [reply]
might want to update the Readme with this info

**Reply by SIX10** [reply]
Will there ever be an update to this, or is this just how it is? I am surprised Mozilla removed this, especially when similar addons like YouTube Redux are allowed. Is there not an appeals process?

**Reply by dimdenGD** [reply]
I already submitted it for review again, waiting for response
